.github/workflows/ci.yml

```diff
@@ -209,11 +232,14 @@ jobs:
       - name: Download test report
         uses: actions/download-artifact@d3f86a106a0bac45b974a628896c90dbdf5c8093 # v4.3.0
         with:
-          pattern: test-report
+          pattern: test-report-linux
+
+      - name: List downloaded files
+        run: ls -la
 
       - name: Zip test report
         run: zip -r test-report.zip *
-        working-directory: test-report
+        working-directory: test-report-linux
```

Comment:
OK for now. Let's see if we need to store reports for other platforms with the releases, too. Ideally it should not be necessary.